environments/azure/flyte-core/ingress-tls.tf

```diff
@@ -0,0 +1,74 @@
+ resource "helm_release" "ingress-nginx" {
+   name             = "ingress-nginx"
+   namespace        = "ingress"
+   create_namespace = true
+   repository       = "https://kubernetes.github.io/ingress-nginx"
+   chart            = "ingress-nginx"
+   depends_on       = [helm_release.flyte-core]
+   set{
+    name = "controller.service.annotations.service\\.beta\\.kubernetes\\.io/azure-load-balancer-health-probe-request-path"
+    value = "/healthz"
+   }
+    provisioner "local-exec" {
+      command = "./scripts/connect_flyte.sh ${azurerm_resource_group.flyte.name} ${azurerm_kubernetes_cluster.flyte.name} ${local.flyte_domain_label}"
+ }
+ }
+
+resource kubernetes_namespace "cert_manager_ns"{
```

Comment:
Is this the only resource coming from the kubernetes provider? If yes, we could also build the namespace with the kubectl provider and ending up with -1 provider